index.js

```diff
@@ -1,7 +1,6 @@
 const emailLog = require("./email_log.json");
 
 function countUniqueEmails(emailFile) {
-  // do work here
   return emailFile.emails.reduce((obj, curr) => {
```

Comment:
I would suggest using meaningful naming conventions for `obj` and `curr`. In this case, `obj` is the previous email object, so you can use something like `prevEmail` and `currEmail`.